src/reloader.cpp

```diff
@@ -234,8 +234,8 @@ void Reloader::reload_indexes(const std::vector<unsigned int>& indexes, bool una
 		ctrl->get_feedcontainer()->unread_item_count();
 
 	partition_reload_to_threads([&](unsigned int start, unsigned int end) {
+		CurlHandle easyhandle;
```

Comment:
There might be an issue in reusing this between different requests if the configured options are not reset.
It will probably work fine for retrieving regular feeds because we use `curl_easy_reset()` at the end of `Parser::parse_url()`.
However, we never reset the configured options in the external APIs.
That could lead to conflicting (or unwanted) headers on subsequent request.

I think this can be resolved by calling `curl_easy_reset(easyhandle.ptr());` after every iteration.

Reply:
I've added the reset call now